src/stories/Tabs/Tabs.tsx

```diff
@@ -22,11 +22,16 @@ export const Tabs = ({
                             onClick={props.handleClick}
                             className="is-active"
                             key={index}
+                            data-testid={'Tab' + index}
                         >
                             <a id={index.toString()}>{element}</a>
                         </li>
                     ) : (
-                        <li onClick={props.handleClick} key={index}>
+                        <li
+                            onClick={props.handleClick}
+                            key={index}
+                            data-testid={'Tab' + index}
```

Comment:
Is `data-testid` attribute absolutely required? As per the [testing library docs](https://testing-library.com/docs/queries/bytestid/):
```
Using data-testid attributes do not resemble how your software is used and should be avoided if possible. That said, they are way better than querying based on DOM structure or styling css class names. 
```
See comment below for an alternative suggestion ;)

Reply:
I added the test ids because the test for which vo tab is active couldn't get the attribute using parent element of the getByText was preventing the webui from compiling

Reply:
@gmatthews20 you can access the parent element without adding the `data-testid` attribute as follows:

```
const atlasTab = screen.getByText('ATLAS')
expect( atlasTab.parentNode.getAttribute('class')).toEqual('is-active')
```
![image](https://user-images.githubusercontent.com/5906242/214089731-286d9c44-1ecf-4cc5-b6cd-d02299bd7b3f.png)

Introducing `data-testid` makes the core codebase dependent on the testing framework, which is counter-intuitive to https://github.com/rucio/webui/issues/117 ;)

Reply:
@maany I have changed from using the testid to get parentElement but it is causing the eslint check to error:
`error    Avoid direct Node access. Prefer using the methods from Testing Library  testing-library/no-node-access`

Reply:
Apologies I did not the suggested change would trigger the compilation error. Let's revert back to data-test-id and if I find something better while working on #117, I will open a follow-up issue ;)